db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :hostname
+      t.string :status
+      t.boolean :scheduling_disabled
+      t.string :scheduling_disabled_reason
+      t.string :zonename
```

Comment:
@blomquisg I agree with you, it should be removed as should hostname based on your assessment.  If it *has* to stay, then based on the [comment in the other thread](https://github.com/ManageIQ/manageiq/pull/7996#discussion-diff-60135441), I think this should be `availability_zone_name`.